crates/sui-graphql-rpc/src/types/move_object.rs

```diff
@@ -23,7 +23,7 @@ pub(crate) struct MoveObject {
 #[Object]
 impl MoveObject {
     // TODO: This depends on having a module resolver so make more efficient
-    async fn contents(&self, ctx: &Context<'_>) -> Result<Option<MoveValue>, Error> {
+    pub async fn contents(&self, ctx: &Context<'_>) -> Result<Option<MoveValue>, Error> {
```

Comment:
This is a bit of an anti-pattern -- it means that you're using the GraphQL API within itself, when usually there is a more efficient way to do it (especially with the introduction of DataLoaders).